### PR TITLE
Moves VerticalCardListFeature component to ui-connected packages

### PR DESCRIPTION
**Pull request by burkeshartsis**
## DESCRIPTION
Companion PR to https://github.com/ApollosProject/apollos-templates/pull/34. Merge this PR first.

### What does this PR do, or why is it needed?
This just packages the `VerticalCardListFeature` into `ui-connected`.

### How do I test this PR?
open the app. check the stories. run the tests.

Actually, tests on this PR wont run because of some mocking error in ui-connected that needs to be resolved. This PR is unmergable until it's resolved.
### What automated tests did you write?
👍 
## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android
- [x] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-ui-connected/src/VerticalCardListFeature/index.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+import {
+  FeedView,
+  H2,
+  H5,
+  PaddedView,
+  styled,
+  withIsLoading,
+} from '@apollosproject/ui-kit';
+import { contentCardComponentMapper } from '@apollosproject/ui-connected';
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/VerticalCardListFeature/index.js:13]: nitpick; This can be a local import now

Comment (burkeshartsis) [reply packages/apollos-ui-connected/src/VerticalCardListFeature/index.js:13]: Completely agree. I just missed it. I'll go back and update all of these deps when I move the final "Features" component
